README.md

```diff
@@ -261,7 +261,9 @@ To actively develop or debug this plugin you can either work directly within the
    This repo includes a fully working, self-seeding instance of Payload that installs the plugin directly from the source code. This is the easiest way to get started. To spin up this demo, follow these steps:
 
    1. First clone the repo
-   1. Then, `cd YOUR_PLUGIN_REPO && yarn && cd demo && cp env.example .env && yarn && yarn dev`
+   1. Then, `cd YOUR_PLUGIN_REPO && yarn && cd demo && cp .env.example .env && yarn`
+   1. Start the database - You can use docker `cd YOUR_PLUGIN_REPO/demo && docker compose up` or a locally installed mongodb instance.
```

Comment:
Need to capitalize the plural nouns here, but also bullet 2 already instructs the user to `cd` into the demo directory so no need for that.
```suggestion
   1. Start the database - You can either use Docker `docker compose up` or a locally installed MongoDB instance
```